bin/commands/generate.mjs

```diff
@@ -131,6 +132,8 @@ export default {
    * @returns {Promise<void>}
    */
   async action(opts) {
+    const logger = Logger.init();
```

Comment:
What's the differenc eof Logger.init() and Logger.getInstance()? Why both exist and what happens if I call Logger.getInstance() before Logger.init()?